### PR TITLE
Fix finding existing Needs label comments

### PR DESCRIPTION
**Pull request by kit1980**
After https://github.com/pytorch/pytorch/pull/101747

### CODE REVIEW

**Review by izaitsevfb — APPROVED** [review]

**Review by malfet — APPROVED** [review]
Is this because title can not have a markup?

File: .github/scripts/label_utils.py [hunk 1]
```diff
@@ -111,7 +111,9 @@ def has_required_labels(pr: "GitHubPR") -> bool:
 
 
 def is_label_err_comment(comment: GitHubComment) -> bool:
+    # comment.body_text returns text without markdown
+    no_format_title = LABEL_ERR_MSG_TITLE.replace("`", "")
```
Comment (zou3519) [inline .github/scripts/label_utils.py:115]: Is there a way to test this? I'm assuming we shipped https://github.com/pytorch/pytorch/pull/101747 and then something happened?

**Review by zou3519 — APPROVED** [review]

**Comment by kit1980** [comment]
> Is this because title can not have a markup?

Not related to title, `GitHubComment(body_text=` just don't return the markup.

File: .github/scripts/label_utils.py (hunk 1 above)
Comment (kit1980) [reply .github/scripts/label_utils.py:115]: After the previous PR landed, I noticed a PR with 2 of this comments, while there should be only one.
There is a unit test that failing after this change, I'll update it.

**Pull request closed without merging**